test/solidity_dynamic_strings.js

```diff
@@ -0,0 +1,51 @@
+const assert = require("assert");
+const pretest = require("./helpers/pretest_setup");
```

Comment:
We can destructure `setup` here.

Reply:
I was opting for readability on this one. Fortunately this is the only export in pretest_setup for now.

Reply:
I'll just rename `setup` to some thing more descriptive. 👍